lib/apiservers/engine/backends/container.go

```diff
@@ -755,7 +755,7 @@ func (c *Container) imageExist(imageID string) (storeName string, err error) {
 	return host, nil
 }
 
-func (c *Container) getImageMetadataFromStoragePL(image string) (*viccontainer.VicContainer, error) {
+func (c *Container) getImageMetadataFromImageC(image string) (*viccontainer.VicContainer, error) {
```

Comment:
Why is this still needed?  The persona has an image cache that should have the image information you seek...